src/main/java/org/example/AppConfig.java

```diff
@@ -0,0 +1,18 @@
+package org.example;
```

Comment:
adapt the project structure to the new structure on main, change org.example to how it is on main